src/app/components/toolbar/base-toolbar/terra-base-toolbar.component.ts

```diff
@@ -16,8 +17,17 @@ export class TerraBaseToolbarComponent
     @Input()
     public inputIsBreadcrumbs:boolean;
```

Comment:
Sollten wir das ganze nicht vllt gleich in die Terra-breadcrumbs.component umziehen statt hier die toolbar zu ergänzen / umzubiegen!?